src/wire/causal_snapshot.rs

```diff
@@ -111,11 +118,18 @@ impl<T: AsRef<[u8]>> WireCausalSnapshot<T> {
         }
     }
 
-    /// Return the `count` field
+    /// Return the `epoch` field
     #[inline]
-    pub fn count(&self) -> u32 {
+    pub fn probe_epoch(&self) -> u16 {
         let data = self.buffer.as_ref();
-        le_bytes::read_u32(&data[field::COUNT])
+        le_bytes::read_u16(&data[field::PROBE_EPOCH])
+    }
+
+    /// Return the `ticks` field
+    #[inline]
+    pub fn probe_ticks(&self) -> u16 {
```

Comment:
Based on the setter method names, seems like this should be named `ticks()`.
Same with `probe_epoch()`, ought to be `epoch()`.